docs/self-managed/deployment/helm/cloud-providers/kind.md

```diff
@@ -0,0 +1,466 @@
+
+You can also use the deployment readiness check script from the root directory of the `camunda-deployment-references` repository. This script requires [jq](https://jqlang.github.io/jq/) to be installed:
+
+```bash
+export CAMUNDA_NAMESPACE=camunda
+```
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/kind-local/generic/kubernetes/single-region/procedure/check-deployment-ready.sh
+```
+
+Finally, verify the Helm release:
+
+```bash
+helm list -n camunda
+```
+
+## Accessing Camunda 8
+
+<Tabs groupId="mode" defaultValue="domain" queryString values={[
+{label: 'Domain mode', value: 'domain'},
+{label: 'No-domain mode', value: 'no-domain'}
+]}>
+
+<TabItem value="domain">
+
+| Component           | URL                                            |
+| ------------------- | ---------------------------------------------- |
+| Operate             | https://camunda.example.com/operate            |
+| Tasklist            | https://camunda.example.com/tasklist           |
+| Identity            | https://camunda.example.com/identity           |
+| Management Identity | https://camunda.example.com/managementidentity |
+| Optimize            | https://camunda.example.com/optimize           |
+| Zeebe REST API      | https://camunda.example.com/                   |
+| Keycloak            | https://camunda.example.com/auth               |
+
+</TabItem>
+
+<TabItem value="no-domain">
+
+Start port-forwarding to access the services:
+
+```bash reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/kind-local/local/kubernetes/kind-single-region/procedure/port-forward.sh
+```
+
+:::tip Localhost development with kubefwd
+For a richer localhost experience, and to avoid managing many individual port-forward commands, you can use [kubefwd](https://github.com/txn2/kubefwd) to forward all services in the target namespace and make them resolvable by their in-cluster DNS names on your workstation. This requires `sudo` to bind privileged ports and modify `/etc/hosts`:
+
+```shell
+sudo kubefwd services -n "camunda"
+```
+
+Now, you can reach services directly, for example:
+
+- **Identity**: `http://camunda-identity/managementidentity`
+- **Keycloak**: `http://camunda-keycloak`
+- **Zeebe Gateway gRPC**: `camunda-zeebe-gateway:26500`
+
+You can still use localhost ports if you prefer traditional port-forwarding. Stop kubefwd with **Ctrl+C** when finished. Be aware kubefwd modifies your `/etc/hosts` temporarily, then restores the file when it exits.
+:::
+
+| Component            | URL                                       |
+| -------------------- | ----------------------------------------- |
+| Zeebe Gateway (gRPC) | localhost:26500                           |
+| Zeebe Gateway (HTTP) | http://localhost:8080/                    |
+| Operate              | http://localhost:8080/operate             |
```

Comment:
With camunda-keycloak in `/etc/hosts`, I no longer get the `ERR_NAME_NOT_RESOLVED`. But now, I get a 404 😅:

```
{"type":"about:blank","title":"Not Found","status":404,"detail":"No message available","instance":"/auth/realms/camunda-platform/protocol/openid-connect/auth"}
```

Also, when I try to access Console, it fails to load the resource. Is there another missing step here, perhaps?